Kitodo/src/main/java/org/kitodo/config/Parameters.java

```diff
@@ -0,0 +1,688 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.config;
+
+/**
+ * These constants define configuration parameters usable in the configuration
+ * file. This file reflects the order of the global
+ * {@code kitodo_config.properties} file (as far as possible), and vice versa.
+ */
+public class Parameters {
+
+    /*
+     * =========================================================================
```

Comment:
Please remove this `===` and `---` lines as they made it unreadable.

Reply:
This is caused because you wanted mltiline comments for it. What causes them to be auto-formatted. It was very clearly readable before, comparing with `kitodo_config.properties`:

![equal](https://user-images.githubusercontent.com/3040657/42328080-b7c7572e-806d-11e8-9113-63a9c8fc1a20.PNG)